packages/rum-core/src/domain/foregroundContexts.spec.ts

```diff
@@ -190,6 +214,49 @@ describe('foreground context', () => {
         })
       })
     })
+
+    describe(`after reaching the maximum number of focus periods: ${MAX_NUMBER_OF_FOCUSED_TIME}`, () => {
+      beforeEach(() => {
+        // given
```

Comment:
not super fond of this types of comments, IMO you can achieve the same with spacing between the different phases.